tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -358,7 +372,7 @@
     "env = environ()\n",
     "\n",
     "# directories for input atom files\n",
-    "env.io.atom_files_directory = ['.']\n",
+    "env.io.atom_files_directory = ['.'], [./nAChR_model’]\n",
```

Comment:
This doesn't seem like correct python to me at all...

I think it should be:
```suggestion
    "env.io.atom_files_directory = ['.', './nAChR_model']\n",
```

Reply:
oof - yes - will correct.